bin/i18n/i18n_script_utils.rb

```diff
@@ -7,6 +7,7 @@
 require 'psych'
 
 I18N_SOURCE_DIR = "i18n/locales/source"
+I18N_SOURCE_DASHBOARD_DIR = File.join(I18N_SOURCE_DIR, 'dashboard').freeze
```

Comment:
Move this out of script utils and into the module as per [this PR](https://github.com/code-dot-org/code-dot-org/pull/52621) comment.